userspace/libsinsp/events/sinsp_events.h

```diff
@@ -159,7 +159,7 @@ class set
 		set<ppm_type> ret(m_max);
 		for(size_t i = 0; i <= m_max; ++i)
 		{
-			if (m_types[i] ^ other.m_types[i])
+			if (m_types[i] == 1 && other.m_types[i] == 0)
```

Comment:
@FedeDP this the issue I found on `libsinsp::events::set`. Set difference is not a commutative operation!

Reply:
ahh i see now :/ they are not bits! Thank you jason!